boards/nxp/mcx_nx4x_evk/board.c

```diff
@@ -18,14 +18,16 @@
 #define BOARD_USB_PHY_TXCAL45DM (0x07U)
 
 usb_phy_config_struct_t usbPhyConfig = {
-	BOARD_USB_PHY_D_CAL, BOARD_USB_PHY_TXCAL45DP, BOARD_USB_PHY_TXCAL45DM,
+	BOARD_USB_PHY_D_CAL,
+	BOARD_USB_PHY_TXCAL45DP,
+	BOARD_USB_PHY_TXCAL45DM,
 };
 #endif
 
 /* Board xtal frequency in Hz */
-#define BOARD_XTAL0_CLK_HZ                        24000000U
+#define BOARD_XTAL0_CLK_HZ    24000000U
```

Comment:
Not a blocking comment but in the future, separate the format changes in a standalone commit from any functional changes. 

For folks who walk through the commits to review them independently, it is easier on their time to see a commit that says it is just format changes with out any functional changes.